src/Harald.WebApi/Features/Connections/Infrastructure/Persistence/FindConnectionsByClientTypeClientIdChannelTypeChannelIdHandler.cs

```diff
@@ -13,24 +15,43 @@ public class FindConnectionsByClientTypeClientIdChannelTypeChannelIdHandler : IQ
         FindConnectionsByClientTypeClientIdChannelTypeChannelId, IEnumerable<Connection>>
     {
         private readonly ICapabilityRepository _capabilityRepository;
+        private readonly ISlackFacade _slackFacade;
 
         public FindConnectionsByClientTypeClientIdChannelTypeChannelIdHandler(
-            ICapabilityRepository capabilityRepository)
+            ICapabilityRepository capabilityRepository,
+            ISlackFacade slackFacade
```

Comment:
Can't run tests in `src/Harald.Tests/features/Connections/Infrastructure/Persistence/FindConnectionsByClientTypeClientIdChannelTypeChannelIdHandlerTests.cs` due to this change of parameters.